wagtailfontawesome/wagtail_hooks.py

```diff
@@ -4,8 +4,12 @@
 from django.conf import settings
 from django.contrib.staticfiles.templatetags.staticfiles import static
 
-from wagtail.wagtailcore import hooks
-from wagtail.wagtailcore import __version__ as WAGTAIL_VERSION
+try:
+    from wagtail.core import hooks
+    from wagtail import __version__ as WAGTAIL_VERSION
+except ImportError:
```

Comment:
Let's add a comment here after we merge, @alexgleason, for our own sanity! :)  `# fallback for Wagtail <2.0`

Reply:
Done